warn/warn_docstring_test.go

```diff
@@ -0,0 +1,238 @@
+package warn
```

Comment:
We've had lots of complains inside Google about Skylint that the docstring parser was too strict or had unclear error messages, e.g. when there was an indentation problem.
Can you check that we don't have the same problems again?